alpha_challenges/the_basics.md

```diff
@@ -143,8 +149,9 @@ end
 **Answer 11:**
 ```ruby
 def names
-  # write your code in here
+  puts ["Jackie", "Hami"]
 end
+return names
```

Comment:
In Ruby the last value inside the function will automatically be returned, so you don't need an explicit return statement. Also when you do have a return statement, it should always be inside a function 😄 

I.e. instead of:
```
def happy
    return 'happy'
end
```

You can just have: 
```
def happy
    'happy'
end
```

Reply:
Thanks for the feedback! Can you show me how it would look like if I included the return inside a function?